packages/hops/mixins/analyzer/mixin.core.js

```diff
@@ -0,0 +1,28 @@
+const { Mixin } = require('hops-mixin');
+
+module.exports = class AnalyzerCoreMixin extends Mixin {
+  configureCommand(definition) {
+    if (definition.command === 'build') {
+      definition.builder.analyzeClientBundle = {
+        type: 'boolean',
+        default: false,
+        describe: 'Opens webpack bundle analyzer page after client side build',
+      };
+    }
+  }
+  handleArguments(argv) {
+    this.options = { ...this.options, ...argv };
+  }
+  configureBuild(webpackConfig, loaderConfigs, target) {
+    if (
+      !['build', 'develop'].includes(target) ||
+      !this.options.analyzeClientBundle
+    ) {
+      return;
+    }
+    const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
+      .BundleAnalyzerPlugin;
+
+    webpackConfig.plugins.push(new BundleAnalyzerPlugin());
```

Comment:
I'm also not a big fan of opening the report directly in the browser, because we don't do that with our server either (as opposed to cra).
So I'd say we should set `openAnalyzer` to false.